JungleBus/Configuration/ReceiveConfigurationExtensions.cs

```diff
@@ -137,6 +137,33 @@ public static IConfigureEventReceiving SetNumberOfPollingInstances(this IConfigu
         }
 
         /// <summary>
+        /// Configure the number of times to attempt to process a message
+        /// </summary>
+        /// <param name="configuration">Configuration to modify</param>
+        /// <param name="count">Number of times to attempt to process a message</param>
+        /// <returns>Modified configuration</returns>
+        public static IConfigureEventReceiving SetMessageRetryCount(this IConfigureEventReceiving configuration, int count)
+        {
+            if (count < 1 || count > 1000)
+            {
+                throw new JungleBusConfigurationException("messageRetryCount", "Number of retries must be between 1 and 1000");

```

Comment:
Updated parameter name.

In looking more at how the code works here, I don't think my change will actually work as is - the retry count is being used before this is set. The simple fix would be to put this as a parameter on SetInputQueue instead (with a default value) - this would be non-breaking to the existing API. A more complex fix is to create the SQS queue as a separate step -> Something like CreateInputQueue - but this would break the API.